src/dispatch.c

```diff
@@ -4,14 +4,21 @@
  * All rights reserved.
  */
 
+#include <unistd.h>
+
+#include "twin_backend.h"
 #include "twin_private.h"
 
-void twin_dispatch(void)
+extern twin_backend_t g_twin_backend;
+
+void twin_dispatch(twin_context_t *ctx)
 {
     for (;;) {
         _twin_run_timeout();
         _twin_run_work();
-        if (!_twin_run_file(_twin_timeout_delay()))
+        if (!g_twin_backend.poll(ctx)) {
```

Comment:
Here TWIN invokes the `poll` function. In [backend/fbdev.c](https://github.com/alanjian85/mado/blob/main/backend/fbdev.c), you also need to implement a `poll` function for the fbdev backend. Such as:
```c
static bool twin_fdev_poll(twin_context_t *ctx)
{
      /* Here are your implementation */
}

/* Register the Linux framebuffer backend */

const twin_backend_t g_twin_backend = {
    .init = twin_fbdev_init,
    .configure = twin_fbdev_configure,
    .poll = twin_fdev_poll,
    .exit = twin_fbdev_exit,
};
```

Reply:
The `twin_linux_input_create()` function already creates a pthread for each input device. The thread has an infinite loop that continuously checks the `/dev/event` files and invokes the event handler whenever an event is available. Therefore, it may not be appropriate to utilize a simple poll function interface for backends with multiple devices, such as fbdev, because this requires keeping track of each registered device.

Reply:
That being said, the fbdev backend only creates an input device in `twin_fbdev_init()`, so it may be feasible to use a singleton-like design to simplify the implementation of `linux_input.c`.

Reply:
Remember to test the framebuffer backend, as Mado can't close properly in your branch.

Reply:
> Remember to test the framebuffer backend, as Mado can't be stopped manually in your branch.

I defer to @shengwen-tw for confirmation.

Reply:
Fixed.